src/components/DataTable/index.jsx

```diff
@@ -1,19 +1,29 @@
-import { Component } from 'react';
+import { Component, Fragment } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import Table from '@material-ui/core/Table';
 import TableBody from '@material-ui/core/TableBody';
 import TableCell from '@material-ui/core/TableCell';
 import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
 import TableSortLabel from '@material-ui/core/TableSortLabel';
-import { arrayOf, func, string, oneOf, object } from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import FilterVariantIcon from 'mdi-react/FilterVariantIcon';
+import Toolbar from '@material-ui/core/Toolbar';
```

Comment:
Move `Toolbar` on top of `FilterVariantIcon` so that `material-ui` packages are grouped together.